.circleci/config.yml

```diff
@@ -108,12 +107,32 @@ jobs:
           command: python setup.py bdist_wheel
       - store_artifacts:
           path: dist
+      - save_cache:
+          key: pip-dist-v1-{{ arch }}
+          paths:
+            - ~/nupic.torch/dist
 
+  deploy:
+    working_directory: ~/nupic.torch
+    docker:
+      - image: circleci/python:3.6
+    steps:
+      - restore_cache:
+          key: source-cache-{{ .Environment.CIRCLE_SHA1 }}
+      - restore_cache:
+          key: pip-dist-v1-{{ arch }}
+      - run:  pip install twine --user
+      - run: echo "DEPLOY TIME"
+      # - run:
+      #     name: Deploy to PYPI
+      #     environment:
+      #       VERBOSE: 1
+      #     command: python -m twine upload --repository-url https://test.pypi.org/legacy/ dist/*
 
 workflows:
   version: 2
 
-  validate-build-test:
+  validate-build-test-deploy:
```

Comment:
If you change the job name then you need to update gitbub branch protection rules:  See https://github.com/numenta/nupic.torch/settings/branch_protection_rules/5520686
I recommend not changing the name if possible.

Reply:
this may be my whole problem